### PR TITLE
[MRG] adapt code for VCF metadata model

### PR DESCRIPTION
**Pull request by gedankenstuecke**

### CODE REVIEW

**Comment by gedankenstuecke** [comment]
The tool now checks the uploaded VCF to see whether its valid and subsequently uploads additional metadata. if it fails it deletes the uploaded file and emails the user.

File: main/celery.py [hunk 1]
```diff
@@ -49,241 +32,76 @@
...
+            base_filename = base_filename[0:-4]
+        meta_filename = base_filename + '.metadata.json'
+        raw_filename = temp_join(tmp_directory, meta_filename)
         metadata = {
                     'description':
-                    '23andMe full genotyping data, original format',
-                    'tags': ['23andMe', 'genotyping'],
-                    'creation_date': arrow.get().format(),
-            }
+                    'VCF file metadata',
```
Comment (madprime) [inline main/celery.py:82]: For consistency with the current legacy source, I think the description would be `"User-contributed VCF data"` - see https://www.openhumans.org/api/public-data/?source=vcf_data

**Review by madprime — APPROVED** [review]
I tested locally to see it all works (i.e. I didn't try to break it) - and it does. :)

Just one very minor comment, and LGTM!

File: main/celery.py (hunk 1 above)
Comment (madprime) [reply main/celery.py:82]: This comment was mistaken! The issue I saw was the description for the main file, not the metadata file - and that's set by admin.

**Pull request merged**